pkg/backup/backup/backup_manager.go

```diff
@@ -576,6 +630,85 @@ func (bm *backupManager) ensureBackupPVCExist(backup *v1alpha1.Backup) (string,
 	return "", nil
 }
 
+// canLogBackupCommandSkipSync returns whether log backup subcommand can skip sync.
+func canLogBackupCommandSkipSync(bm *backupManager, backup *v1alpha1.Backup) (bool, error) {
```

Comment:
you can break function to small one more readable, if you organize like this:
1. if already start return true
2. if already truncate return true
3. if already stop return true 
4. else return false.

Reply:
ok, that will be more readable, i will fix it.